package.json

```diff
@@ -92,6 +92,7 @@
     "test:watch": "TZ=UTC vitest",
     "test:coverage": "TZ=UTC vitest run --coverage",
     "build": "fec build",
+    "build:dev": "webpack --config webpack.config.js",
```

Comment:
Can we rename cockpit related commands so it's obvious? Like "cockpit:build" etc.